VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -275,13 +275,12 @@ public void onSuccess(AuthResponse result) {
                             }
                         } catch (LoginException ex) {
                             super.onException(ex);
+                            onUserLoginFailure(ex, null, null);
                         }
                     }
-
                     @Override
                     public void onException(Exception ex) {
                         super.onException(ex);
-                        onUserLoginFailure(ex, null, null);
```

Comment:
It wouldn't matter. https://github.com/edx/edx-app-android/blob/master/VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java#L462

If it's not a LoginException, it won't be handled the way we want it to be.